tests/OmniSharp.Roslyn.CSharp.Tests/BlockStructureFacts.cs

```diff
@@ -0,0 +1,58 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using OmniSharp.Models;
+using OmniSharp.Models.GotoFile;
+using OmniSharp.Roslyn.CSharp.Services.Navigation;
+using OmniSharp.Roslyn.CSharp.Services.Structure;
+using TestUtility;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OmniSharp.Roslyn.CSharp.Tests
+{
+    public class BlockStructureFacts : AbstractSingleRequestHandlerTestFixture<BlockStructureService>
+    {
+        public BlockStructureFacts(ITestOutputHelper output, SharedOmniSharpHostFixture sharedOmniSharpHostFixture)
+            : base(output, sharedOmniSharpHostFixture)
+        {
+        }
+
+        protected override string EndpointName => OmniSharpEndpoints.BlockStructure;
+
+        [Fact]
+        public async Task UsesRoslynBlockStructureService()
+        {
+            var testFile1 = new TestFile("foo.cs", @"class Foo
+{
+    void M()
+    {
+        if (false)
+        {
+        }
+    }
+}");
+            var testFile = new TestFile("bar.cs", @"class Bar {}");
+
+            var lineSpans = (await GetResponseAsync(testFile1)).
+                    Select(l => (l.Line, l.EndLine)).ToArray();
+
+            var expected = new [] { (0, 8), (2, 7), (4, 6) };
```

Comment:
There is a markup feature for tests in OmniSharp that is similar to Roslyn's that you can use instead of raw numbers. You can see it used in places like this: https://github.com/OmniSharp/omnisharp-roslyn/blob/master/tests/OmniSharp.Roslyn.CSharp.Tests/FormattingFacts.cs#L141